scripts/setup-centos9.sh

```diff
@@ -191,23 +191,29 @@ ARROW_VERSION=15.0.0
 
 function install_arrow {
   wget_and_untar https://archive.apache.org/dist/arrow/arrow-${ARROW_VERSION}/apache-arrow-${ARROW_VERSION}.tar.gz arrow
-  cd arrow/cpp
-  cmake_install \
-    -DARROW_PARQUET=OFF \
-    -DARROW_WITH_THRIFT=ON \
-    -DARROW_WITH_LZ4=ON \
-    -DARROW_WITH_SNAPPY=ON \
-    -DARROW_WITH_ZLIB=ON \
-    -DARROW_WITH_ZSTD=ON \
-    -DARROW_JEMALLOC=OFF \
-    -DARROW_SIMD_LEVEL=NONE \
-    -DARROW_RUNTIME_SIMD_LEVEL=NONE \
-    -DARROW_WITH_UTF8PROC=OFF \
-    -DARROW_TESTING=ON \
-    -DCMAKE_INSTALL_PREFIX=/usr/local \
-    -DCMAKE_BUILD_TYPE=Release \
-    -DARROW_BUILD_STATIC=ON \
-    -DThrift_SOURCE=BUNDLED
+  (
+    cd arrow/cpp
+    cmake_install \
+      -DARROW_PARQUET=OFF \
+      -DARROW_WITH_THRIFT=ON \
+      -DARROW_WITH_LZ4=ON \
+      -DARROW_WITH_SNAPPY=ON \
+      -DARROW_WITH_ZLIB=ON \
+      -DARROW_WITH_ZSTD=ON \
+      -DARROW_JEMALLOC=OFF \
+      -DARROW_SIMD_LEVEL=NONE \
+      -DARROW_RUNTIME_SIMD_LEVEL=NONE \
+      -DARROW_WITH_UTF8PROC=OFF \
+      -DARROW_TESTING=ON \
+      -DCMAKE_INSTALL_PREFIX=/usr/local \
+      -DCMAKE_BUILD_TYPE=Release \
+      -DARROW_BUILD_STATIC=ON \
+      -DThrift_SOURCE=BUNDLED
+
+    # Install thrift.
+    cd _build/thrift_ep-prefix/src/thrift_ep-build
+    $SUDO cmake --install ./ --prefix /usr/local/
```

Comment:
$SUDO isn't defined here. Don't you also need to add the definition in this script like it is done for Ubuntu (and in the functions that call cmake etc)? It isn't inherited from `setup-helper-functions.sh`. 
On Ubuntu it does this:
```
SUDO="${SUDO:-"sudo --preserve-env"}"
```

Nit: do we want to use `${SUDO}` like in the other scripts?

Reply:
@czentgr, I just removed the use of ${SUDO} in this setup-centos9.sh and ${SUDO} is still not defined with `sudo`, consistent with other places where `cmake_install` is used in this script.
BTW, I also find `sudo` is not installed in some centos-9 images used for velox CI. So just defining ${SUDO} without installing `sudo` can cause execution failure.